prody/proteins/interactions.py

```diff
@@ -718,8 +718,9 @@ def calcPiCation(atoms, **kwargs):
             sele1 = atoms.select('resid '+str(i[0])+' and chain '+i[1]+' and '+aromatic_dic[sele1_name[0]])
             sele2 = atoms.select('(same residue as exwithin '+str(distA)+' of center) and resname ARG LYS and noh and not backbone and not name NE "C.*"', 
                                center=calcCenter(sele1.getCoords()))
-        except:
-            LOGGER.info("Missing atoms from the side chains of the structure. Use PDBFixer.")
+        except ImportError:
```

Comment:
no, this is not an import error for the except because you didn't try to import anything. There could be an index error or a dictionary key error or an error from calcCenter not having anything in sele1

Reply:
That was fine, yes? (below)
        except:
            LOGGER.info("Missing atoms from the side chains of the structure. Use PDBFixer.")
 I will change it here and in other places.

Reply:
Can you still make it an error not a LOGGER.info? Otherwise, yes, I guess it's ok. 

You could do
```
except:
    ValueError("Missing atoms from the side chains of the structure. Use addMissingAtoms.")
```

Reply:
Ok. I will change that.